src/scripts/main.js

```diff
@@ -18,11 +18,21 @@ const food = {
 
   },
 };
-
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const list = document.createElement('ul');
+
+  for (const key in data) {
+    if (typeof (data[key]) === 'object') {
+      const part = document.createElement('li');
+
+      part.innerHTML = key;
```

Comment:
Using `part.innerHTML = key;` may expose your code to security risks (XSS attacks). Instead, use `part.textContent = key;` to safely set the text content of the list item 🛡️